src/simulation/dynamics/prescribedMotion/_UnitTest/test_PrescribedMotionStateEffector.py

```diff
@@ -179,14 +179,13 @@ def PrescribedMotionTestFunction(show_plots, rotTest, thetaInit, theta_Ref, posI
         # Initialize the prescribedRot1DOF test module configuration data
         accelMax = 0.01  # [rad/s^2]
         #accelMax = np.pi / 180  # [rad/s^2]
+        PrescribedRot1DOF.coastOption = False
```

Comment:
Why is this required?  Doesn't `coastOption` default to False?

Reply:
Currently, the user is required to set the `coastOption` variable. Should I change this?

Reply:
Yes, I would make this variable an option to set.  If not set, the module would run as before.

Reply:
Done